src/week.jsx

```diff
@@ -41,6 +42,12 @@ export default class Week extends React.Component {
     }
   }
 
+  handleWeekClick = (day, weekNumber, event) => {
+    if (this.props.onWeekSelect) {
```

Comment:
1) it is a little weird that this condition the same as L67, there is probably a way to just have one condition
2) I think that since we then call onWeekSelect the condition could check that onWeekSelect is a function. I know that you have checked it in the propTypes anyway but I think that it would be more cohesive to check it here. (plus the prop type warning with only give a warning if onWeekSelect is defined as something other than a function, whereas here it would actually cause a 'function not defined' error on L47